server/server.go

```diff
@@ -38,5 +37,21 @@ func main() {
 		return c.JSON(http.StatusOK, res)
 	})
 
+  
+  type Content struct {
+    Username string `json:"username"`
+    ContentID string `json:"id"`
+    ContentBody string `json:"content_body"`
+  }
+
+  e.POST("/postTest", func(c echo.Context) (err error) {
```

Comment:
We shouldn't call this postTest -- this is meant to be our actual production content creation endpoint.